style.css

```diff
@@ -271,43 +355,214 @@ li {
+    width: 140px;
+  }
+}
+
+@media screen and (max-width: 480px) {
+  /* couldn't find any bootstrapp class for this  */
   #logo {
-    width: 150px;
+    width: 100px;
+  }
+}
+
+.image-top-story-container {
+  height: 300px;
+  position: relative;
+}
+
+.article-heading {
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 16px;
+}
+
+/* couldn't find any bootstrapp class for this  */
+
+.top-story-category {
+  bottom: 0;
+  left: 0;
+  background-color: #ccc;
+  color: var(--custom-primary-color);
+  font-weight: bold;
+}
+
+/* couldn't find any bootstrapp class for this  */
+
+.top-story-category:hover {
+  color: var(--custom-primary-color);
+}
+
+.top-article-details-container p {
+  color: rgb(51, 51, 51);
+  line-height: 20px;
+  margin-top: 10px;
+}
+
+/* couldn't find any bootstrapp class for this  */
+
+.culture-category {
+  bottom: 0;
+  left: 0;
+  color: #555;
+  background-color: #ccc;
+  z-index: 99;
+  font-size: 14px;
+  font-weight: 500;
+}
+
+.culture-category:hover {
+  color: #555;
+}
+
+.story-article .story-image {
+  height: 90px;
+  width: 95px;
+  margin-right: 15px;
+}
+
+.small-logo div {
+  width: 30px;
+  height: 30px;
+}
+
+.story-article .story-image div {
+  height: 95px;
+  width: 95px;
+}
+
+.story-text p {
+  font-size: 14px;
+}
+
+/* couldn't find any bootstrapp class for this  */
+
+.feature {
+  height: 220px;
+}
+
+/* couldn't find any bootstrapp class for this  */
+
+.image-top-story-container.feature.inside {
+  border: 1px solid #333;
+}
+
+/* couldn't find any bootstrapp class for this  */
+
+.signup {
+  background: var(--custom-primary-color);
+}
+
+.signup-box {
+  background-color: rgba(204, 204, 204, 0.274);
+  padding: 20px;
+  margin-bottom: 40px;
+}
+
+.small-logo {
+  background-color: var(--custom-primary-color);
+  width: 30px;
+  height: 30px;
+  margin-right: 10px;
+}
+
+.small-logo-text {
+  font-size: 17px;
+  font-weight: 500;
+}
+
+.supply p {
+  font-size: 13px;
+  font-weight: bolder;
+  color: var(--custom-gray-color);
+  line-height: 1;
+}
+
+.supply {
+  margin-left: 10px;
+  flex: 1;
+}
+
+/* couldn't find any bootstrapp class for this  */
+
+.debate-main-area {
+  border-top: 4px solid rgba(165, 163, 163, 0.562);
+  border-bottom: 4px solid rgba(165, 163, 163, 0.562);
+  padding: 15px 0;
+  margin-bottom: 40px;
+  position: relative;
+}
+
+/* couldn't find any bootstrapp class for this  */
+
+.debate-vs {
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+  background-color: var(--custom-search-nav-color);
+  top: 40%;
+  left: 44%;
+}
+
+/* couldn't find any bootstrapp class for this  */
+
+.my-turn-img.middle {
+  height: 100px;
+}
+
+/* couldn't find any bootstrapp class for this  */
+
+@media screen and (max-width: 1000px) {
+  .debate-container {
+    margin: 0 auto;
+  }
+
+  .signup-box {
+    margin: 0 auto;
+  }
+}
+
+/* couldn't find any bootstrapp class for this  */
+
+@media screen and (max-width: 768px) {
+  .debate-container {
+    margin: 0 auto;
```

Comment:
You have used `@media screen and (max-width: 768px)` so many times for different classes. See line no. 586 and 334.